sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveCatalogs.scala

```diff
@@ -80,8 +80,8 @@ class ResolveCatalogs(val catalogManager: CatalogManager)
         writeOptions = c.writeOptions,
         orCreate = c.orCreate)
 
-    case UseStatement(isNamespaceSet, nameParts) =>
-      if (isNamespaceSet) {
+    case Use(UnresolvedDBObjectName(nameParts, isNamespace)) =>
```

Comment:
shall we have a single place to resolve `UnresolvedDBObjectName`? Currently it's resolved in L34 in this file

Reply:
@cloud-fan, Yeah, but I didn't see an other way to keep USE working as the same as UseStatement with ResolvedDBObjectName, should we create new JIRA for this issue?

Reply:
One way to resolve `UnresolvedDBObjectName` in single place, is

`case UnresolvedDBObjectName(CatalogAndNamespace(catalog, name), _) =>
      ResolvedDBObjectName(catalog, name)
......
  case Use(ResolvedDBObjectName(catalog, ns)) =>
      val namespace = if (ns.nonEmpty) Some(ns) else None
      SetCatalogAndNamespace(catalogManager, Some(catalog.name()), namespace)
`

But it will change the behavior of `USE NAMESPACE ...` to keep it in line with `CREATE NAMESPACE ...`, which is NO different with `USE ...`.